### DISCUSSION TITLE
[keyball44] 特定のキーが深く押下しないと反応しない

### DISCUSSION THREAD

**Discussion by ghost** [Yowkees/keyball#489] [Q&A] [answered]
![image](https://github.com/Yowkees/keyball/assets/67816657/72c660fc-b724-4f78-864c-e810607f3b00)
画像赤丸のCtrlになります。ソケットもスイッチも別のものにしてみましたが解消せずです。

こういった場合に確認すべきことや解消方法があれば教えてください。

**Comment by ghost** [comment]
訂正
深くというより長く押さないと反応しませんでした
一瞬だけパっと何回か押しても反応しませんが押し続けるとちょっと遅れて反応します

**Comment by ghost** [comment, marked as answer]
remapで確認したところ、langとctrlになってたのですね。
単押しではlangだったためテスターに反応せずに勘違いをしていたようです。
失礼しました。クローズします

**Comment by Yowkees** [comment]
ご自身で解決いただけたようで良かったです。
ぜひ末永くご愛用いただければ幸いです。
